lib/xebow/animation.ex

```diff
@@ -0,0 +1,96 @@
+defmodule Xebow.Animation do
+  alias Xebow.RGBMatrix
+
+  @callback init_state(pixels :: list(RGBMatrix.pixel())) :: t
```

Comment:
@doughsay do you have opinions on `list()` versus `[]` in the typespec? I noticed you had been using `list()` so I went with that in this PR. However, if you don't have a preference, my preference is towards `[]` since it's the same syntax as an Elixir list and therefore less conversion for my brain to do.

```ex
@callback init_state(pixels :: [RGBMatrix.pixel()]) :: t
```

Reply:
I don't have a strong preference.  I use `list()` because it's similar to `map()` unless you have a similar argument for maps?

Reply:
Same preference with `map` for me. I tend to lean towards the [literals](https://hexdocs.pm/elixir/typespecs.html#literals) when it comes to typespec formatting.

Reply:
:thinking: Actually, I lie a little. I tend to deviate from the literals when it comes to `binary` vs. `<<>>`. Tangential to this PR, I should try to articulate my preferences into style guidelines so they're more clear.

Reply:
I'll defer this topic for another time. I don't think I should block the PR over it and can proceed with `list()`.